lib/psulib_traject/call_number.rb

```diff
@@ -102,6 +105,10 @@ def dewey?
         classification == 'DEWEY'
       end
 
+      def sudoc?
+        classification == 'SUDOC'
```

Comment:
I like these shortcuts.

Reply:
Is there any possibility the returned case would ever be different (eg 'Sudoc')? I assume not, but just wanted to check.

Reply:
Hmm, good question.  I would say no since the others don't do any case normalization.  It's probably some standard for marc that they are always in uppercase, but I wouldn't be against adding case normalization to all of these to be safe.